### PR TITLE
[change]: Paths for mapping

### PR DESCRIPTION
**Pull request by seramg**

### CODE REVIEW

File: src/pages/authTeachers/Show.tsx [hunk 1]
```diff
@@ -32,10 +32,15 @@ const AuthorizedTeacherShow = () => {
...
-            setClassroomData(Object.values(e.data.classrooms));
-            setSubjectData(Object.values(e.data.classrooms).map((f) => f.subject));
-        });
+        dataProvider
+            .getOne<Teacher>(MAPPING.TEACHERS, { id: authorizedTeacher?.id })
+            .then((e) => {
+                setClassroomData(Object.values(e.data.classrooms));
+                setSubjectData(Object.values(e.data.classrooms).map((f) => f.subject));
+            })
+            .catch(() => {
```
Comment (mak626) [inline src/pages/authTeachers/Show.tsx]: We should not display console.log in prod. Is this an error that breaks prod?

File: src/provider/mapping.ts
Comment (mak626) [inline src/provider/mapping.ts:1]: this should be dynamic right in some way. We need to bring some configuration system ig

**Review by mak626 — CHANGES_REQUESTED** [review]
We need to discuss the mapping configuration management system @amaljoe

File: src/pages/authTeachers/Show.tsx (hunk 1 above)
Comment (seramg) [reply src/pages/authTeachers/Show.tsx]: hey, please look into the new commit

**Review by amaljoe — APPROVED** [review]

**Pull request merged**